code/game/supplyshuttle.dm

```diff
@@ -335,6 +372,18 @@ var/datum/controller/supply/supply_controller = new()
 	var/points_per_slip = 1
 	var/points_per_crate = 2
 
+	//black market stuff
+	///in Weyland-Yutani dollars - Not Stan_Albatross.
```

Comment:
why is satanbatross referenced in code...

Reply:
that was what he put in his unfinished black market branch i based this on (minus the not)

Reply:
huh